py/desitarget/gfa.py

```diff
@@ -29,7 +29,8 @@
 #ADM the current data model for columns in the GFA files
 gfadatamodel = np.array([], dtype=[
     ('TARGETID', 'i8'),  ('BRICKID', 'i4'), ('BRICK_OBJID', 'i4'),  
-    ('RA', 'f8'), ('DEC', 'f8'), ('TYPE', 'S4'),
+    ('RA', 'f8'), ('DEC', 'f8'), ('RA_IVAR', 'f8'), ('DEC_IVAR', 'f8'), 
```

Comment:
In the sweeps the RA/DEC IVARS are dtype f4 not f8.  Let's also keep them as f4 here.

Note: RA and DEC themselves are f8, which is needed.  This is only a comment on the IVARs.